examples/python/depth_guided_stable_diffusion/huggingface_pipeline.py

```diff
@@ -208,8 +208,8 @@ def _encode_prompt(self, prompt, device, num_images_per_prompt, do_classifier_fr
                 if `guidance_scale` is less than `1`).
         """
         batch_size = len(prompt) if isinstance(prompt, list) else 1
-        rr.log("prompt/text", rr.TextLog(prompt))
-        rr.log("prompt/text_negative", rr.TextLog(negative_prompt))
+        rr.log("prompt/text", rr.TextDocument(prompt))
+        rr.log("prompt/text_negative", rr.TextDocument(negative_prompt))
```

Comment:
I found text log to be better looking for those single lines because then in the blueprint I can put both text prompts in the same space view, making everything a bit more compact. In any case this doesn't work out with the blueprint now anymore.